### DISCUSSION TITLE
파트 분배 관련

### DISCUSSION THREAD

**Discussion by ccmuk58** [The-Day-In-1984/1984_Project#19] [General]
Discussions 익힐 겸 작성해봅니다.

https://github.com/The-Day-In-1984/1984_Project/blob/main/Proceedings/230313.md

- 플랫포머(맵, 애니메이션)
- 임무(미션, 평판게이지 시스템)
- 스토리(대화)

저번 회의 때 이야기가 나왔던 파트 분배 부분입니다.
각자 희망하는 파트 있으실까요?

저 같은 경우 전 프로젝트에서 플레이어블 기능을 주로 작업해서, 약간이나마 플랫포머 이외의 파트를 진행하고 싶습니다.
물론 해도 전혀 상관없긴 합니다.

의견 주시면 감사하겠습니다!

**Comment by fkdl0048** [comment]
먼저 의견 내주셔서 감사합니다..! 

~~까먹고 있었습니다..~~

저는 정말 상관 없습니다..  

공부하고 싶은 부분은 비동기 처리를 해보고 싶어서 그런 부분만 잘 섞을 수 있는 파트면 어디든 상관 없을 것 같아요

@kyoonge 민경님도 프로젝트 진행하시면서 공부하면서 해보고 싶은 파트가 있으면 말씀해주세요!

**Reply by kyoonge** [reply]
호옥시 그럼 제가 플랫포머 쪽 구현 맡아도 될까요?
부족하지만 공부하고 싶어서요!

**Reply by ccmuk58** [reply]
그럼 제가 대화 구현쪽으로 해봐도 될까요?
아직 라이브러리 추가 안됐으면 공부 겸 직접 구현해보고 싶습니다!
안될 것 같으면 나중에 라이브러리 사용하는 걸로 ,,,

**Reply by fkdl0048** [reply]
좋습니다..! 제가 임무쪽이랑 전체적인 시스템 설계쪽 담당할께요
